eclipsePlugin/build.gradle

```diff
@@ -397,10 +399,34 @@ task siteStableLatestXml(type:Copy) {
   }
 }
 
+// sign all .jar file under specified dir
+ext.signJar = { File dir ->
+  def keystorepass = project.hasProperty('keystorepass') ? keystorepass : '';
+  if (keystorepass.isEmpty()) {
+    print 'to sign eclipse plugins, set "keystorepass" project property'
```

Comment:
So I see this message while building - what do I need to do to? Where is this need to be set to which value?

Reply:
To sign in local, we need to [create Java KeyStore file](https://docs.oracle.com/javase/tutorial/security/toolsign/step3.html) first. We can generate self-signed one by:

```sh
$ keytool -genkey -alias eclipse-plugin -keystore $PROJECT_ROOT/spotbugs.jks -storepass changeit
```

Then use `-Pkeystorepass` option or `gradle.properties` to set project properties:

```sh
$ ./gradlew clean eclipseSite -Pkeystorepass=changeit
```

Reply:
To release our eclipse plugin, we use not self-signed keystore file but signed by Let's Encrypt.
But I think self-signed one is enough to test this change.

Reply:
OK, with the given explanation the signing works locally and installed plugin is accepted and shown as signed. But this means, we should update "how to build Ecipse plugin" manual / readme.md, because it is really not obvious.